src/chrome/content/rules/Chefkoch.de.xml

```diff
@@ -0,0 +1,16 @@
+<ruleset name="Chefkoch (partial)">
+
+	<target host="*.chefkoch.de" />
+	<target host="*.chefkoch-cdn.de" />
+	<target host="*.chefkoch-blog.de" />
+
+	<rule from="^http://(css|emskr|img|js|static)\.chefkoch-cdn\.de/"
+		to="https://$1.chefkoch-cdn.de/" />
+		
+	<rule from="^http://cdn\.chefkoch\.de/"	
+		to="https://chefkoch.de/" />
```

Comment:
I mean

		<rule from="^http://cdn\.chefkoch\.de/"
		        to="https://static.chefkoch-cdn.de/" />

because cdn.chefkoch.de and static.chefkoch-cdn.de are both cdn domains (IPs 185.13.208.24, 185.13.208.25) but chefkoch.de isn't (IP 185.13.208.22 only).